scripts/build.sh

```diff
@@ -69,7 +69,7 @@ echo "..... Trying to install the new tarball inside a virtualenv"
 # note: installs with the optional dependency to verify this is still working
 virtualenv .venv/test-tarball
 source .venv/test-tarball/bin/activate
-pip install --no-binary :all: -f dist/ pylint_django[with_django]
+pip install -f dist/ pylint_django[with_django]
```

Comment:
The reason for this line & the reason why it is failing is the same - it tries installing pylint-django and all of its dependencies from tartballs. One of them is either new or stopped shipping tarballs, and only has wheel packages. See 10 lines below for how we deal with another package which doesn't ship wheels.

My preferred way for dealing with this is to find which package is the one not providing tarballs and
1) Open an issue against it
2) Install it explicitly by itself.

Reply:
For now I'll accept this in our own CI while I track down which dependency is the guilty party - I am fairly sure it's clikit and I'll raise a PR/issue there if that's true or as you suggest, explicitly install it ourselves.

Reply:
I created a new issue to separate this out: https://github.com/PyCQA/pylint-django/issues/291